src/promptflow/promptflow/azure/operations/_flow_operations.py

```diff
@@ -282,9 +289,31 @@ def _create_remote_flow_via_file_share_path(
         )
         return rest_flow_result
 
-    def _get(self, flow_id):
-        # TODO: support load remote flow with meta
-        raise NotImplementedError("Not implemented yet")
+    def get(self, name: str) -> Flow:
+        """Get a flow from azure.
+
+        :param name: The name of the flow to get.
+        :type name: str
+        :return: The flow.
+        :rtype: ~promptflow.azure.entities.Flow
```

Comment:
promptflow.entities.Flow

Reply:
Offline synced, I have opened a task 2819200 to track this.